Formula/mypy.rb

```diff
@@ -2,8 +2,8 @@ class Mypy < Formula
   desc "Experimental optional static type checker for Python"
   homepage "http://www.mypy-lang.org/"
   url "https://github.com/JukkaL/mypy.git",
-    :tag => "v0.4.2",
-    :revision => "b8a2df91707bc3d5c9c3677c8017b614e41d017a"
+      :tag => "v0.4.3",
```

Comment:
There's also a discrepancy in the indentation of multi-line system command invocations. For configure, `brew create` produces
```
    system "./configure", "--disable-debug",
                          "--disable-dependency-tracking",
                          "--disable-silent-rules",
                          "--prefix=#{prefix}"
```

However, for `go` formulae, when a system command invocation wraps, the "correct" way is apparently:
(d)
```
    system "go", "build", "-o", bin/"slackcat",
           "-ldflags", "-X main.version=#{version}"
```
or
(e)
```
 system "go", "build",
        "-o", bin/"slackcat",
        "-ldflags", "-X main.version=#{version}"
```
but not
(f)
```
    system "go", "build", "-o", bin/"slackcat",
                          "-ldflags", "-X main.version=#{version}"
```

I'm not sure what accounts for this discrepancy and which is actually our preferred style. If it is indeed (d), it seems the `brew create` template should be changed accordingly, and non-compliant formulae updated.